changelogs/CHANGELOG-1.17.md

```diff
@@ -0,0 +1,139 @@
+## v1.17
+
+### Download
+https://github.com/vmware-tanzu/velero/releases/tag/v1.17.0
+
+### Container Image
+`velero/velero:v1.17.0`
+
+### Documentation
+https://velero.io/docs/v1.17/
+
+### Upgrading
+https://velero.io/docs/v1.17/upgrade-to-1.17/
+
+### Highlights
```

Comment:
Could you also mention the extension of resource policy to add include/exclude policies?

https://github.com/vmware-tanzu/velero/pull/9210/files#diff-ed333e58353834216293fb2562aed92f6eb492bb0107ebe9c94a862e33e68e33